src/main/scala/beam/router/r5/R5RoutingWorker.scala

```diff
@@ -280,8 +281,8 @@ class R5RoutingWorker(val beamServices: BeamServices, val transportNetwork: Tran
           //TODO make a more sensible window not just 30 minutes
           startTime >= time.fromTime && startTime <= time.fromTime + 1800
         }.map(itinerary => {
-          var legsWithFares = Vector[(BeamLeg, Double)]()
-          maybeWalkToVehicle.foreach(legsWithFares +:= (_, 0.0))
+          val legsWithFares = ArrayBuffer[(BeamLeg, Double)]()
```

Comment:
How about just move it into a one line initializer?

`val legsWithFares = maybeWalkToVehicle.map(x=> ArrayBuffer((x, 0.0))).getOrElse(ArrayBuffer[(BeamLeg, Double)]())`